app/views/curation_concerns/base/_relationships_collection_row.html.erb

```diff
@@ -0,0 +1,16 @@
+<tr>
+<th><%= t(".#{model_name}", type: model_name) %></th>
+<td>
+  <% if collection.blank? %>
+    <p><%= t('.empty', type: model_name) %></p>
+  <% else %>
+    <ul class="tabular">
+      <% collection.each do |col| %>
+        <li class='attribute title'>
+          <%= link_to col.title.first, contextual_path(col, nil) %>
```

Comment:
When the partial is rendered in the context of a `Collection` it doesn't have a parent presenter like a regular work would. I was expecting `contextual_path(col)` to eventually just call `polymorphic_path([col])` but maybe there needs to be a fix in CC to check `parent_presenter.nil?` to make this all flow nicely without us needing to pass a `nil` in?

https://github.com/projecthydra/curation_concerns/blob/a1fdf2d2deedf089f2ba52dc8a03208f26e0b344/app/services/curation_concerns/contextual_path.rb#L12